scripts/symbolize.py

```diff
@@ -104,6 +104,9 @@ def get_elf(self, elf_or_uuid):
     def set_arch(self):
         if self._arch:
             return
+        self._arch = os.getenv('CROSS_COMPILE', '<none>');
+        if not self._arch is '<none>':
+            return
```

Comment:
ok here? (please crosscheck syntax and all, i'm definitely not good at python)

Reply:
Yep, looks fine.

Reply:
```
self._arch = os.getenv('CROSS_COMPILE');
if self._arch:
    return
```
should achieve the same effect. `os.getenv` defaults to `None` if the env variable is not found and we return if `self._arch` is not `None` after the `os.getenv` function call.

Reply:
@Emantor indeed, much better ;-) @etienne-lms please use this form and apply my R-b.

Reply:
@Emantor, thanks for the proposal.

Reply:
Well, I was to update with this proposal but contrary to the previous, it fails if `CROSS_COMPILER` is defined but empty. I saw that using `export CROSS_COMPILE=` allow to get the native `nm`, `objdump`, ... of the environment. With the proposal, defined but empty `CROSS_COMPILE` are treated as if `CROSS_COMPILE` is not defined.

I don't know what is the best :(